streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -2479,7 +2502,7 @@ public void shouldTransmitProducerMetrics() {
             allTasks.put(task.id(), (StateMachineTask) task);
         }
         for (final Task task : restoringTasks) {
-            assertThat(task.state(), not(Task.State.RUNNING));
```

Comment:
Nothing really changed here, but we use to just leave tasks in CREATED and call them "restoring" so I had to fix this up so they really were RESTORING